README.md

```diff
@@ -1,2 +1,4 @@
+[![CircleCI](https://circleci.com/gh/AndelaOSP/art-dashboard/tree/ch-integrate-circle-ci-157340050.svg?style=svg)](https://circleci.com/gh/AndelaOSP/art-dashboard/tree/ch-integrate-circle-ci-157340050)
+
```

Comment:
The badge integrates to the branch specifically. I think it should be directed to the default branch.